mod.d/arpcache.yaml

```diff
@@ -16,41 +16,43 @@
 # Module document. Translates directly into an almost-complete Module object
 name: !!str arpcache
 path: !!str
-version: !!str 1.0
+version: !!str 2.0
 title: !!str Determines if aggressive arp caching is enabled
 helptext: !!str |
   Determines if aggressive arp caching is enabled.
   This can cause problems communicating with same subnet resources
 placement: !!str run
 package: 
   - !!str
-language: !!str bash
+language: !!str python
 content: !!str |
-  #!/bin/bash
-  error_trap()
-  {
-      printf "%0.s=" {1..80}
-      echo -e "\nERROR:	"$BASH_COMMAND" exited with an error on line ${BASH_LINENO[0]}"
-      exit 0
-  }
-  trap error_trap ERR
+  from __future__ import print_function
+  import re
+  import subprocess
 
-  # read-in shared function
-  source functions.bash
+  try:
+      print("Determining if agressive ARP caching is is enabled")
+      arpstatus = subprocess.check_output(["sysctl", "net.ipv4.neigh.default.gc_thresh1"]).decode("utf-8")
+      if re.search("0", arpstatus):
+          arp_success = True
+      else:
+          arp_success = False
+      if arp_success:
```

Comment:
This if block seems redundant. The prints could be moved into the preceding if regex block.